lib/ui/Popup.js

```diff
@@ -199,13 +200,13 @@ Popup.prototype = {
         var imgMatches = chromeImgLink.match(/^moz-anno\:favicon\:(.*)/);
 
         results.push({
-          url: Components.classes["@mozilla.org/intl/texttosuburi;1"].
+          url: Components.classes['@mozilla.org/intl/texttosuburi;1'].
                 getService(Components.interfaces.nsITextToSubURI).
-                unEscapeURIForUI("UTF-8", controller.getValueAt(i)),
+                unEscapeURIForUI('UTF-8', controller.getValueAt(i)),
           image: imgMatches ? imgMatches[1] : null,
           title: controller.getCommentAt(i),
           type: controller.getStyleAt(i),
-          text: controller.searchString.replace(/^\s+/, "").replace(/\s+$/, "")
+          text: controller.searchString.replace(/^\s+/, '').replace(/\s+$/, '')
```

Comment:
I'm not sure what the regex is happening here. Not sure if we can just use this to strip the leading/trailing whitespaces:
```js
text: controller.searchString.trim()
```